zephyr/src/supp_mgmt.h

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
So, is this really different from Zephyr's wifi mgmt layer?

Reply:
It's a copy of the wifi_mgmt.h but uses a different _NET_SUPP_CODE to avoid conflict with the wifi_mgmt enums. We are going update this to include the full WPA_CLI commands that supplicant supports going forward.

Reply:
I still strongly encourage to try to extend the actual wifi mgmt layer from Zephyr when we have a clear image on what is needed.

Reply:
As per offline discussions. This has now been removed, shell directly talks to wpa_supplicant code.

Reply:
The header should also be removed then?